### ISSUE TITLE
tagline addition

### ISSUE THREAD

**Issue by thinq4yourself** [BeardandFedora/writeon.io#13]
Hand crafted for writers.

Passionately crafted for writers.

Etc. Idea came from #chagirl

**Comment by yaboi** [comment]
So, the complete tagline would be "The World's Finest Writing Experience Hand Crafted For Writers" or "The World's Finest Writing Experience Passionately Crafted For Writers" ?

I like the idea and direction, but think it's a little verbose. Maybe rewording the whole tagline to be something like "A Hand Crafted Writing Experience." Either that, or working "crafted" and "passion" into secondary tag lines, similar to the "Write. Publish. Period." secondary tag lines.

#ChaGirlIsYaGirl

**Comment by thinq4yourself** [comment]
New additions to site and marketing pages (angel.co, etc):

> On a mission to craft the world's finest writing experience"
> 
> Passionately Crafted For Writers
> Especially the picky ones like us.
